lib/modules/manager/kotlin-script/index.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { regEx } from '../../../util/regex';
+import { defaultConfig } from './index';
+
+describe('modules/manager/kotlin-script/index', () => {
+  it('fileMatch regex is correct', () => {
+    defaultConfig.fileMatch.forEach((pattern) => {
```

Comment:
```suggestion
    expect(defaultConfig.fileMatch).toHaveLength(1);
    defaultConfig.fileMatch.forEach((pattern) => {
```

normally we don't test default config, but if you do, do it right 😉